package.json

```diff
@@ -2,17 +2,28 @@
   "name": "react-directus",
   "version": "0.0.2",
   "description": "A set of React components and utilities for Directus Headless CMS",
+  "homepage": "https://github.com/gremo/react-directus",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gremo/react-directus.git"
+  },
+  "type": "module",
   "files": [
```

Comment:
Changing the files array isn't really necessary for "common files". Try the following:

```bash
npm run build
npm pack
```

In the tarball, by default, you have `LICENSE`, `package.json` and `README.md`.

Any reason to include `src` and `tsconfig.json`?

Reply:
You're right, by default, the tarball includes LICENSE, package.json, and README.md.

As for including `src` and `tsconfig.json`, I wanted to make it a bit more explicit for users who might be interested in the source code and TypeScript configuration. It can also facilitate modifications to the package, and I find it helpful in some edge cases where users may want to recompile the project themselves.

Let me know if you think it's better to exclude them, and I'll be happy to make the necessary adjustments based on your preference.

Thank you for bringing this up, and I appreciate your input on the package structure!

Reply:
AFAIK, you can google it, the best practice for packaging for NPM is to only include files for runtime. In fact, now one will digg into node_modules folder and look for src or tsconfig.json.

Reply:
Thank you for pointing that out. I agree that including `tsconfig.json` and the `src` folder might not be necessary for most users, but it's far from uncommon, as demonstrated by [Directus SDK](https://www.npmjs.com/package/@directus/sdk?activeTab=code)

While there are cases where having these files can be useful for debugging or patching, I understand that it's not a common use case for most users. To keep the package minimal, I will go ahead and remove `tsconfig.json` and `src` from the PR.